transforms/helpers/ast.ts

```diff
@@ -400,8 +392,8 @@ export function makeEOActionInfiniteLiteralAssertion(
 function isEOActionInfiniteLiteral(
```

Comment:
What is an infinite literal? 😲

Reply:
LOL.

IIRC this is for detecting if transforming an action object property into an `@action` method on the class would cause an infinite loop.

e.g. I think something like this was a thing:

```
export default Foo = EmberObject.extend({
  doSomething() {},
  
  actions: {
    doSomething() {
      this.doSomething()
    }
  }
});
```

But it would would transform to:

```
@classic
export default class Foo extends EmberObject {
  doSomething() {}

  doSomething() {
    this.doSomething()
  }
}
```

which is wrong in multiple ways.